src/runtime/server/middleware/session/sessionStorage.ts

```diff
@@ -1,5 +1,8 @@
 import { createStorage } from 'unstorage'
+// @ts-ignore
 import sessionDriver from '#session-driver'
-import sessionConfig from '#session-config'
+import { useRuntimeConfig } from '#imports'
+const config = useRuntimeConfig()
+const sessionConfig = config.session
```

Comment:
the problem here is that this is invoked outside of a composable. Composables can only be invoked inside a client-side components or other composable, but not outside of it, e.g., here.

I don't know if this was the problem with your general approach, just leeting you know.

I'm on vacay, so @valiafetisov / @zoey-kaiser will carry this over the line with you (:

Reply:
Apparently you can access [runtimeConfig](https://v3.nuxtjs.org/api/composables/use-runtime-config#acess-runtime-config) composables outside of a component. I'm guessing this one works outside of a component because, I'm assuming, its not reactive.

Reply:
Hm, weird, I had a different memory of these docs, thanks for clearing this up 🙏